release/models/interfaces/openconfig-interfaces.yang

```diff
@@ -1048,10 +1055,56 @@ module openconfig-interfaces {
 
     leaf carrier-transitions {
       type oc-yang:counter64;
+      status deprecated;
       description
         "Number of times the interface state has transitioned
         between up and down since the time the device restarted
-        or the last-clear time, whichever is most recent.";
+        or the last-clear time, whichever is most recent.
+
+        Please use interface-transitions instead, which has
+        similar, but more precisely specified, semantics and a
+        clearer name.";
+      oc-ext:telemetry-on-change;
+    }
+
+    leaf interface-transitions {
+      type oc-yang:counter64;
+      description
+        "The total number of times the interface state (oper-status)
+        has either transitioned to 'UP' state from any other state, or
+        from state 'UP' to any other state.  I.e., an interface flap
+        from UP to DOWN back to UP increments the counter by 2.
+        Transitions between any other interface states other than to
+        or from 'UP' state are not included in the counter.
+
+        Discontinuities in the value of this counter can occur
+        when the device is re-initialization as indicated by the
+        value of 'last-clear'.";
```

Comment:
re-initialized iso re-initialization?
Does this discontinuity note also apply to link-transitions?

Reply:
Good spot.  I just directly copied this text from in-fcs-errors.  I will fix in both places.